recipe/meta.yaml

```
@@ -6,8 +6,11 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/openai/atari-py/archive/{{ version }}.zip
-  sha256: 2ab97bc59f196f91d1b2227ac1cece3ef3fda996d9acf7a8c93e866d4bdc9f3f
+  - url: https://github.com/openai/atari-py/archive/{{ version }}.zip
+    sha256: 2ab97bc59f196f91d1b2227ac1cece3ef3fda996d9acf7a8c93e866d4bdc9f3f
+  - url: https://zlib.net/zlib1211.zip  # [win]
+    sha256: d7510a8ee1918b7d0cad197a089c0a2cd4d6df05fee22389f67f115e738b178d  # [win]
+    folder: atari_py/ale_interface/src/zlib  # [win]
```

My personal preference is to align selectors, but just as a comment, not a request.
```suggestion
  - url: https://zlib.net/zlib1211.zip                                        # [win]
    sha256: d7510a8ee1918b7d0cad197a089c0a2cd4d6df05fee22389f67f115e738b178d  # [win]
    folder: atari_py/ale_interface/src/zlib                                   # [win]
```